static/repos.json

```diff
@@ -2,46 +2,55 @@
   {
     "name": "apiary",
     "description": "Apiary provides modules which can be combined to create a federated cloud data lake",
-    "imageUrl": "https://repository-images.githubusercontent.com/128752984/e774bf00-80c6-11eb-88db-337cf4a24fda"
+    "imageUrl": "https://repository-images.githubusercontent.com/128752984/e774bf00-80c6-11eb-88db-337cf4a24fda",
+    "repoUrl": "https://github.com/ExpediaGroup/apiary"
```

Comment:
Minor and not sure if it makes sense but can't we reconstruct this to avoid duplication?
It'll always be `https://github.com/ExpediaGroup/` + $name.

Reply:
I also had the same doubt, but in the end I thought that completely delegating the responsibility of creating the repo url to the GitHub API and transparently use the returned field was a more future proof solution.. WDYT?

Reply:
Yes, sure.